src/dev-app/cdk-experimental-listbox/cdk-listbox-demo.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Component} from '@angular/core';
+
+@Component({
+    templateUrl: 'cdk-listbox-demo.html',
+    styleUrls: ['cdk-listbox-demo.css'],
+})
+export class CdkListboxDemo {
+    multiSelectable = false;
```

Comment:
Nit: This file should use 2 spaces for indentation. 

<sup>We should set up a formatter at some point again: #19879</sup>